spec/build/script/d_spec.rb

```diff
@@ -13,7 +13,7 @@
   it_behaves_like 'a build script sexp'
 
   it 'downloads and runs the installer script' do
-    should include_sexp [:cmd, %r{curl.*https://dlang\.org/install\.sh.*\|\|.*curl.*https://nightlies\.dlang\.org/install\.sh}m,
+    should include_sexp [:cmd, %r{curl.*https://dlang\.org/install\.sh.*\|\|.*curl.*https://nightlies\.dlang\.org/install\.sh.*\|\|.*curl.*https://nightlies\.dlang\.org/install\.sh.*\|\|.*curl.*https://raw.githubusercontent.com/dlang/installer.*}m,
```

Comment:
This looks to be wrong. https://travis-ci.org/travis-ci/travis-build/jobs/323798377#L673